### PR TITLE
Add Issue and PR templates

### PR DESCRIPTION
**Pull request by jlee-r7**
See https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+`~/.msf4/logs/framework.log` for relevant stack traces
+
+
+## System stuff
+
+### Metasploit version
+
+Get this with the `version` command in msfconsole (or `git log -1 --pretty=oneline` for a source install).
+
+### I installed Metasploit with:
```
Comment (bcook-r7) [inline .github/ISSUE_TEMPLATE.md]: this will be very helpful, thanks

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## Steps to reproduce
+
+How'd you do it?
+
+1. ...
+2. ...
+
+This section should also tell us any relevant information about the
+environment; for example, if an exploit that used to work is failing,
+tell us the victim operating system and service versions.
```
Comment (jhart-r7) [inline .github/ISSUE_TEMPLATE.md]: I feel like a little more detail here wouldn't hurt.  For example, if they have the name of the module used, that is critical.  If they have the exact steps and output from `msfconsole`, that would be helpful.  If they have logs, great.

Comment (jlee-r7) [inline .github/ISSUE_TEMPLATE.md]: I was trying to balance info with brevity here. If it's too long, folks won't read it. Maybe we should move the stuff about logs down around line 20 up here?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
+Tell us what this change does. If you're fixing a bug, please mention
+the github issue number.
+
```
Comment (OJ) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should we add `## Sample Output` or `## Same Run` here too?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Tell us what this change does. If you're fixing a bug, please mention
+the github issue number.
```
Comment (OJ) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is it worth saying that links to `metasploit-payloads` PRs are required if Meterpreter changes are related?

Comment (bcook-r7) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not a bad idea, though most PRs I've seen for metasploit-payloads have already done this, so not a huge problem.

**Comment by bcook-r7** [comment]
Let's get these in, then we can tweak as we go.

**Pull request merged**